android/quest/src/main/java/org/smartregister/fhircore/quest/ui/questionnaire/QuestionnaireViewModel.kt

```diff
@@ -779,6 +788,23 @@ constructor(
       }
   }
 
+  fun appendPractitionerInfo(resource: Resource) {
```

Comment:
Looks like this will duplicate,  `org.smartregister.fhircore.engine.util.extension.ResourceExtensionKt#appendPractitionerInfo` (my bad). Can you update the implementation to include the missing resources.